pyccl/halos/profiles.py

```diff
@@ -267,17 +275,167 @@ def cumul2d(self, cosmo, r_t, M, a, mass_def=None):
             are scalars, the corresponding dimension will be
             squeezed out on output.
         """
-        if getattr(self, '_cumul2d', None):
+        if getattr(self, "_cumul2d", None):
             s_r_t = self._cumul2d(cosmo, r_t, M, a, mass_def)
         else:
-            s_r_t = self._projected_fftlog_wrap(cosmo, r_t, M,
-                                                a, mass_def,
-                                                is_cumul2d=True)
+            s_r_t = self._projected_fftlog_wrap(
+                cosmo, r_t, M, a, mass_def, is_cumul2d=True
+            )
         return s_r_t
 
-    def _fftlog_wrap(self, cosmo, k, M, a, mass_def,
-                     fourier_out=False,
-                     large_padding=True):
+    def sigma_critical(self, cosmo, a_lens, a_src):
```

Comment:
Wouldn't it be preferable to not use word initials/shorthands? I think renaming `a_src` to `a_source` would read better in a code. Plus, it won't add many extra characters. For example, `a_lens` is not cut to `a_lns`.

Reply:
It seems to me like `sigma_critical` is not really a profile-dependent quantity, but a cosmology-dependent one. I think it'd be more appropriate to define this as a standalone function (e.g. in background.py).

Reply:
Fixed